resources/views/errors/404.blade.php

```diff
@@ -0,0 +1,8 @@
+@extends('errors::minimal')
+
+@section('code', '404')
+@section('message')
+    Lost in the Digital Maze? You've reached a page that's on a coffee break.<br>
```

Comment:
Have to tell the client why they are on this page.
```suggestion
    Lost in the Digital Maze? You've reached a page that does not exist.<br>
```